### PR TITLE
Change title

### PR DESCRIPTION
**Pull request by Maniktherana**

### CODE REVIEW

File: pages/index.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Steps,Callout } from 'nextra/components'
 
-# About ASoC
+# About alias summer of code
```
Comment (vyogami) [inline pages/index.mdx:3]: Capatilization of ALiAS is incorrect!
Can you fix it?

Comment (Maniktherana) [reply pages/index.mdx:3]: sure thing

Comment (vyogami) [reply pages/index.mdx]: Looks good!

**Comment by vyogami** [comment]
You can merge it

**Pull request merged**